lib/spack/spack/test/install.py

```diff
@@ -58,6 +58,25 @@ def fake_fetchify(url, pkg):
 
 
 @pytest.mark.usefixtures('install_mockery')
+def test_source_copy(mock_archive):
```

Comment:
It should run. All functions that start with `test_` should be run. I'm not sure how often Codecov updates, if that's what you're asking.